draft-ietf-tls-tls13.md

```diff
@@ -1890,14 +1894,16 @@ Cookies serve two primary purposes:
 When sending a HelloRetryRequest, the server MAY provide a "cookie" extension to the
 client (this is an exception to the usual rule that the only extensions that
 may be sent are those that appear in the ClientHello). When sending the
-new ClientHello, the client MUST echo the value of the extension.
-Clients MUST NOT use cookies in subsequent connections.
+new ClientHello, the client MUST echo the value of the extension and SHOULD
+subsequently delete the local copy of the cookie value.
+Clients MUST NOT reuse cookie values in subsequent connections.
```

Comment:
What's the point of deleting the cookie? It's not secret because it just went over the wire.